src/main/java/net/snowflake/client/jdbc/SnowflakeFileTransferAgent.java

```diff
@@ -1580,7 +1590,7 @@ private void uploadFiles(Set<String> fileList, int parallel) throws SnowflakeSQL
 
   private void segregateFilesBySize() {
     for (String srcFile : sourceFiles) {
-      if ((new File(srcFile)).length() > BIG_FILE_THRESHOLD) {
+      if ((new File(srcFile)).length() > (multipartUploadThreshold)) {
```

Comment:
This is just a nit pick
```suggestion
      if ((new File(srcFile)).length() > multipartUploadThreshold) {
```

Reply:
Good point, parentheses removed